src/command_modules/azure-cli-servicebus/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+
+0.3.5
++++++
+* added netwrokruleset commands
```

Comment:
Typo. Should say:

* `namespace`: Added `network-rule` commands.
* `namespace create/update`: Added `--default-action` argument for network rules.

Reply:
done